internal/server/utils/redisCmdAdapter.go

```diff
@@ -100,6 +106,13 @@ func ParseHTTPRequest(r *http.Request) (*cmd.RedisCmd, error) {
 						delete(jsonBody, key)
 						continue
 					}
+					if key == Members {
```

Comment:
Can you add some corresponding tests in redisCmdAdapter_test for Member/Offset and Members?

Reply:
Yeah I'll add these